### PR TITLE
fix: Replaced hero image with an optimized version.

### PR DESCRIPTION
**Pull request by shamimbinnur**
## Description
The hero image was causing excessive load. The reason behind this was that the image was not translated properly when exporting it as an SVG from Figma, resulting in a size of 9 MB. I had the draft Figma file, so I tried to re-export it, and with a little compression, I was able to reduce its size from 9 MB to 2.1 MB.

**This PR replaces the hero image with the optimized one.**

Note: The hero image I added to PR has a different avatar than the previous one (I do not have the previous one).
**Before**
Screenshot 2023-09-18 at 11 20 33 PM
**Now**
Screenshot 2023-09-18 at 11 19 58 PM

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by shamimbinnur** [comment]
I would like to request a review @bdougie

File: public/hero.svg
Comment (bdougie) [inline public/hero.svg:1]: @shamimbinnur thanks for improving this. However, the image provided is a different photo than the previous one. Do you have access to the latest one in Figma?

The new version is actually the original and the close up on the face is a bit awkward.

Comment (shamimbinnur) [reply public/hero.svg:1]: @bdougie  Sorry for that. Unfortunately, I do not have access to the latest Figma.

Comment (bdougie) [reply public/hero.svg:1]: I can ask @getheaddev for it.

Comment (shamimbinnur) [reply public/hero.svg:1]: @bdougie Thanks. I just changed the avatar according to the latest design.

**Review by bdougie — APPROVED** [review]

**Pull request merged**